apache/superset
# refactor(explore): Enhance Dataset and Control panel Collapse components

### SUMMARY
Removes react-bootstrap Collapse with Antdesign enhanced Collapse.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
Before:

https://user-images.githubusercontent.com/60598000/105727326-41f8f800-5f2b-11eb-8ae3-bf7ee4d7e866.mp4

After:

https://user-images.githubusercontent.com/60598000/105727342-46bdac00-5f2b-11eb-9826-dae94ae274dd.mp4

### TEST PLAN
1. Open the explorer
2. Make sure the control panel has all the collapsible expanded
3. Close a collapsible panel, the content should be hidden
4. Re-open the closed collapsible panel, the content should appear

### ADDITIONAL INFORMATION
